analyzers/src/SonarAnalyzer.Common/Rules/SymbolicExecutionRunnerBase.cs

```diff
@@ -72,47 +72,47 @@ protected static bool IsEnabled(SonarSyntaxNodeReportingContext context, Diagnos
         }
     }
 
-    protected void Analyze<TNode>(SonarAnalysisContext analysisContext, SonarSyntaxNodeReportingContext context, Func<TNode, SyntaxNode> getBody) where TNode : SyntaxNode
+    protected void Analyze(SonarAnalysisContext analysisContext, SonarSyntaxNodeReportingContext context)
     {
-        if (getBody((TNode)context.Node) is { } body && context.SemanticModel.GetDeclaredSymbol(context.Node) is { } symbol)
+        if (context.SemanticModel.GetDeclaredSymbol(context.Node) is { } symbol)
         {
-            Analyze(analysisContext, context, body, symbol);
+            Analyze(analysisContext, context, symbol);
         }
     }
 
-    protected void Analyze(SonarAnalysisContext analysisContext, SonarSyntaxNodeReportingContext nodeContext, SyntaxNode body, ISymbol symbol)
+    protected void Analyze(SonarAnalysisContext analysisContext, SonarSyntaxNodeReportingContext context, ISymbol symbol)
```

Comment:
The `node` part of `nodeContext` name was important here. Because there's also `analysisContext` and we need to clearly distinguish those two apart.